apps/crn-server/src/queries/events.queries.ts

```diff
@@ -24,6 +24,7 @@ export const eventContentFragment = gql`
       notes
       videoRecordingPermanentlyUnavailable
       videoRecording
+      videoRecordingUpdatedAt
```

Comment:
do we need this field anywhere other than the reminders? If not - I think just adding it to this file should suffice: https://github.com/yldio/asap-hub/pull/2077/files#diff-efb614c8fe87caa1c2764a46bdd6ae5248322d1383f1db077527ba6816223012

Reply:
The `videoRecordingUpdatedAt` is being used in this sort function

```
const getSortDate = (reminder: ReminderDataObject): DateTime => {
  if (reminder.entity === 'Research Output') {
    return DateTime.fromISO(reminder.data.addedDate);
  }

  if (reminder.type === 'Happening Today') {
    return DateTime.fromISO(reminder.data.startDate);
  }

  if (reminder.type === 'Video Updated') {
    return DateTime.fromISO(reminder.data.videoRecordingUpdatedAt);
  }

  return DateTime.fromISO(reminder.data.endDate);
};
```

I believe it's possible to remove the startDate and endDate from the videoReminder though :)